app/views/provider_interface/reports/withdrawal_reports/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<%= content_for :title, t('page_titles.provider.withdrawal_report') %>
+<%= content_for :before_content, breadcrumbs(t('page_titles.provider.reports') => provider_interface_reports_path,
+                                             t('page_titles.provider.withdrawal_report') => nil) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">
+      <span class="govuk-caption-l"> <%= @provider.name %></span>
+      <%= t('page_titles.provider.withdrawal_report') %>
+    </h1>
+  </div>
+</div>
+
+<% if @submitted_withdrawal_reason_count < 10 %>
```

Comment:
suggest the service include a `MIN_SUBMITTED_WITHDRAWAL_REASONS_TO_SHOW = 10`, and use that here as well as in the text below